tldr: update page (#13276)
tldr-pages/tldr: pages/common/tldr.md

- [l]ist all pages for the current platform and `common`:

`tldr --list`

> - [l]ist all available subcommands for the current platform in tldr pages:

Reviewer: ```suggestion
- [l]ist all available subcommand pages for a command:
```

Reviewer: Why you resolved this suggestion?

Author: yesterday I skimmed and mistook resolving it

> - [l]ist all available subcommands for the current platform in tldr pages:
> `tldr --list | grep {{command}} | column`

Reviewer: **
```suggestion
- [l]ist all available subcommands for the current platform in tldr pages:

`tldr --list | grep {{command}} | column`
```
**
